spec/system/project_show_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Project Show Page", stub_mediaflux: true do
+  let(:sponsor_user) { FactoryBot.create(:user, uid: "pul123") }
+  let(:project_not_in_mediaflux) do
+    Project.create(metadata: { directory: "project-111", data_sponsor: "pul123", title: "project 111", departments: [] })
+  end
+
+  let(:project_in_mediaflux) do
+    project = Project.create(metadata: { directory: "project-222", data_sponsor: "pul123", title: "project 222", departments: [] })
+    project.approve!(session_id: sponsor_user.mediaflux_session, created_by: sponsor_user.uid)
```

Comment:
This clarifies much regarding the directional flow for persistence and state relating to `Project` Objects, thank you!